authoring-service/src/test/kotlin/com/devotion/authoring/service/DefaultSurveyAuthoringServiceTest.kt

```diff
@@ -219,10 +217,8 @@ class DefaultSurveyAuthoringServiceTest {
         val SURVEY_ID_3_NON_PUBLISHED_WITH_1_QUESTION = "3"
         val SURVEY_ID_4_NON_PUBLISHED_WITH_NO_QUESTIONS = "4"
 
-        fun getQuestions(size: Int): MutableList<Question> {
-            val result = ArrayList<Question>(size)
-            (0 until size).mapTo(result) { Question(it.toString()) }
-            return result
+        fun getQuestions(size: Int): List<Question> {
```

Comment:
Use expression body instead of block body

Reply:
👎